CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 ### Added
+- `Markdown` components support code highlighting - no need to switch to `SyntaxHighlighter`, which has been removed. Use triple backticks, with the opening backticks followed by the language name or abbreviation. [#562](https://github.com/plotly/dash-core-components/pull/562) Supported languages:
```

Comment:
Maybe add that `Markdown` will attempt to auto-detect the language (via `highlight.js`) without specifying the language name

Reply:
Also maybe mention the `theme` prop, but this could be in the docs for `dcc.Markdown` as well/instead